packages/amplify-e2e-tests/schemas/custom_query.graphql

```diff
@@ -11,7 +11,7 @@ type Comment @model {
   todo: Todo @hasOne
 }
 
-type CommentConnection {
+type commentConnection {
```

Comment:
Do we need to change the reference to this `type` below as well? https://github.com/aws-amplify/amplify-category-api/pull/1291/files#diff-73a9a466ec4f9056241d6a90bae4aec1445c929555121fe15f31581cb518f5c9R20

Reply:
now adding it for code uniformity, but looks like capitalization of first letter doesn't effect after the change :)